src/os/filestore/FileStore.cc

```diff
@@ -1866,7 +1866,7 @@ void FileStore::init_temp_collections()
 
 int FileStore::umount()
 {
-  dout(5) << "umount " << basedir << dendl;
+  dout(5) << __func__ << ": " << basedir << dendl;
```

Comment:
some where there is colon, and other where there is no colon, so I should add colon at all place to keep the same.